lib/matplotlib/quiver.py

```diff
@@ -1009,7 +1009,8 @@ def _make_barbs(self, u, v, nflags, nbarbs, half_barb, empty_flag, length,
 
         *pivot* specifies the point on the barb around which the
         entire barb should be rotated.  Right now, valid options are
-        'head' and 'middle'.
+        'tip' and 'middle'. Can also be a number, which shifts the start
+        of the barb that many units from the origin.
```

Comment:
What units are polygons/paths/markers normally in? I don't think it's pixels, but it's not data coordinates either.